report/controls.go

```diff
@@ -32,3 +36,40 @@ func (cs Controls) Copy() Controls {
 func (cs Controls) AddControl(c Control) {
 	cs[c.ID] = c
 }
+
+// NodeControls represent the individual controls that are valid
+// for a given node at a given point in time.  Its is immutable.
+type NodeControls struct {
+	Timestamp int64  `json:"timestamp"`
```

Comment:
No reason; I'm used to thinking of time like this.  It seems the go way is to use a time.Time, as it has all the right json marshalling baked in.  I can't think of any gotchas, so will change.